bin/truffleruby.sh

```diff
@@ -89,19 +89,32 @@ java_args+=("-XX:MetaspaceSize=25M")
 # Truffle
 binary_truffle="$root/mx.imports/binary/truffle/mxbuild"
 source_truffle="$root_parent/graal/truffle/mxbuild"
+bootcp=()
 if [ -f "$binary_truffle/dists/truffle-api.jar" ]; then # Binary Truffle suite
     truffle="$binary_truffle"
-    graal_sdk="$(dirname "$binary_truffle")/mx.imports/binary/sdk/mxbuild/dists/graal-sdk.jar"
-    launcher_common="$(dirname "$binary_truffle")/mx.imports/binary/sdk/mxbuild/dists/launcher-common.jar"
+    bootcp+=(
+        "$binary_truffle/dists/truffle-api.jar"
+        "$(dirname "$binary_truffle")/mx.imports/binary/sdk/mxbuild/dists/graal-sdk.jar"
+        "$(dirname "$binary_truffle")/mx.imports/binary/sdk/mxbuild/dists/launcher-common.jar"
+        "$(dirname "$binary_truffle")/mx.imports/binary/tools/mxbuild/dists/chromeinspector.jar"
+        "$(dirname "$binary_truffle")/mx.imports/binary/tools/mxbuild/dists/truffle-profiler.jar"
+    )
 elif [ -f "$source_truffle/dists/truffle-api.jar" ]; then # Source Truffle suite
     truffle="$source_truffle"
-    graal_sdk="$root_parent/graal/sdk/mxbuild/dists/graal-sdk.jar"
-    launcher_common="$root_parent/graal/sdk/mxbuild/dists/launcher-common.jar"
+    bootcp+=(
+        "$source_truffle/dists/truffle-api.jar"
+        "$root_parent/graal/sdk/mxbuild/dists/graal-sdk.jar"
+        "$root_parent/graal/sdk/mxbuild/dists/launcher-common.jar"
+        "$root_parent/graal/tools/mxbuild/dists/chromeinspector.jar"
+        "$root_parent/graal/tools/mxbuild/dists/truffle-profiler.jar"
```

Comment:
Those paths are very similar to above, they only differ in the prefix.
Could you define a variable pointing to the relevant repo base (`"$(dirname "$binary_truffle")/mx.imports/binary` or `"$root_parent/graal`) and then deduplicate?